doc/library/graph/op.rst

```diff
@@ -1,12 +1,7 @@
-
 .. _libdoc_graph_op:
 
-==============================================================
-:mod:`graph` -- Objects and functions for computational graphs
-==============================================================
+===========================================
+:mod:`op` -- Objects that define operations
+===========================================
 
 .. automodule:: pytensor.graph.op
```

Comment:
I think `:members:` is missing, this generates https://pytensor--817.org.readthedocs.build/en/817/library/graph/op.html. See https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#usage

Reply:
Oh yes, I deleted `members` by mistake, also above